### ISSUE TITLE
Players not spawning in the level

### ISSUE THREAD

**Issue by eros71-dev** [ipodtouch0218/NSMB-MarioVsLuigi#5]
Found another issue, you probably know about it already but sometimes the players don't spawn at all. You can see the arrow pointing towards the other player but not the other character itself.
![imagen](https://user-images.githubusercontent.com/16540103/159096548-23b111a1-3354-452a-8f9b-ef7aa59f9b29.png)

**Comment by ipodtouch0218** [comment]
What I'm getting from your description is that the other player is invisible, yet the arrow in the UI appears and follows them?

Is there any way to replicate this consistently? Because I've yet to experience this bug myself. If you get it to happen again, upload your Player.log: `C:\Users\<User>\AppData\LocalLow\ipodtouch0218\NSMB-MarioVsLuigi\Player.log`.

**Comment by eros71-dev** [comment]
Sorry for not answering before, I did a cleanup and redownloaded the latest release, everything seems to work for now. If it happens again I'll send the Player.log right away.

**Comment by TheMoogle** [comment]
I was able to reproduce this in webgl, and I have a friend who tried to join me earlier also have this issue
![image](https://user-images.githubusercontent.com/34310114/159416791-a0d66055-b3dc-4b94-9257-c6f2529be629.png)
![image](https://user-images.githubusercontent.com/34310114/159416966-1ba2c837-c3bd-44d2-a30e-b8f45f8c0813.png)

**Comment by TheMoogle** [comment]
Browser console logs [console-export-2022-3-22_0-59-14.txt](https://github.com/ipodtouch0218/NSMB-MarioVsLuigi/files/8321570/console-export-2022-3-22_0-59-14.txt)

**Comment by ipodtouch0218** [comment]
`FileNotFoundException: Could not find file "/idbfs/22f9466b2b8ff2d262322d6415ba2df8/controls.json"`

Seems to be a WebGL issue, as we don't have access to a file system to be able to read/write controls. *(WebGL was never planned to be supported, I'm surprised it even boots... if it works well enough i might put it on itch...)* 

This could also theoretically happen on non-WebGL builds by removing controls.json before the game loads.


Currently, control binding overrides are saved to file and read when loading the player (as overrides are instance-specific, and rebindings didn't stay on scene change). It's possible that just serializing the bindings to a string in the GlobalController and loading on player spawn would also work. 

I'll test it tomorrow.

**Comment by Tinman1775** [comment]
Experiencing the same issue that is originally described, attached is my player.log file. In my case, I haven't spawned in any scenario, by myself or multiplayer. Even after a fresh install of the Windows build I was still running into the same issue
[Player.log](https://github.com/ipodtouch0218/NSMB-MarioVsLuigi/files/8322036/Player.log)

**Comment by PoketrainerMS** [comment]
i dont know how to recreate it but this happened to me too recently when playing the web version on itch, from what i remember, i was tabbed out (which was causing my ping to skyrocket btw, probably bc its a chromebook and i was in a discord call with my friends that i was playing it with) and the game started, and i came back right as it started, and i never spawned. my screen looked exactly how this looked but with the exception of having a different HUD bc i assume its been updated since then
> ![imagen](https://user-images.githubusercontent.com/16540103/159096548-23b111a1-3354-452a-8f9b-ef7aa59f9b29.png)

to everybody else, i was under the floor at spawn kinda like this image (but not webgl obviously)
> ![image](https://user-images.githubusercontent.com/34310114/159416966-1ba2c837-c3bd-44d2-a30e-b8f45f8c0813.png)

basically what im trying to say is that i dont think it was fixed fully, or the bug came back with a recent update
